### PR TITLE
WP-774 Update sample to display Ethiopian calendar

### PR DESCRIPTION
**Pull request by DavidUM**
## Issue

Make timePicker supports Ethiopia time
## Fix

Current timePicker does not consider that meridiem has some entries more than two -AM & PM-
Fix related logic

Enyo-DCO-1.1-Signed-off-by: David Um david.um@lge.com

### CODE REVIEW

File: src/moonstone-samples/lib/TimePickerSample.js [hunk 1]
```diff
@@ -83,7 +84,7 @@ module.exports = kind({
 	},
 	timeChanged: function (sender, event) {
 		if (this.$.result && event.value){
-			var timeArray = event.value.toTimeString().split(': ');
+			var timeArray = event.value.toTimeString().split(':');
```
Comment (aarontam) [inline src/moonstone-samples/lib/TimePickerSample.js]: I don't know if we need to split the time string anymore - can we just display the whole time string, instead?

File: src/moonstone-samples/lib/TimePickerSample.js [hunk 2]
```diff
@@ -83,7 +84,7 @@ module.exports = kind({
...
 			this.$.result.setContent(event.name + ' changed to ' + timeArray[0] + ': ' + timeArray[1]);
```
Comment (aarontam) [inline src/moonstone-samples/lib/TimePickerSample.js]: Alternatively, if we want to display just the hour and minute (as was done in 2.5.x), let's remove the extra space after the `:` when concatenating the strings.

File: src/moonstone-samples/lib/DatePickerSample.js [hunk 3]
```diff
@@ -84,6 +85,7 @@ module.exports = kind({
 			i18n.updateLocale(locale);
 			this.$.picker.setLocale(val);
 			this.$.disabledPicker.setLocale(val);
+			this.$.result.setContent("locale Changed to " + val);
```
Comment (aarontam) [inline src/moonstone-samples/lib/DatePickerSample.js]: For consistency and flexibility, can we change "locale" to the value of `event.originator.name`? Also, let's lowercase the "Changed" to "changed".

**Comment by aarontam** [comment]
Can you reverse the file mode changes (644 -> 755) so that this is not part of the change?

File: src/moonstone-samples/lib/TimePickerSample.js (hunk 1 above)
Comment (DavidUM) [inline src/moonstone-samples/lib/TimePickerSample.js]: Yes you are right. I just keep consistency with 2.5-upkeep.
Actually we don't need split it anymore

**Comment by DavidUM** [comment]
No, I didn't change file mode. I revert it to 644.
I update sample to sync localeValue with result content.

**Pull request merged**